extensions/mini/__init__.py

```diff
@@ -30,6 +30,16 @@ def register(self):
                 "function" : self.ping,
                 "description" : "Ping the bot for a response",
                 "roles" : ["@everyone"]
+            },
+            "color" : {
+                "function" : self.color,
+                "description" : "Obtain the color of a role on the server",
+                "roles" : ["@everyone"]
+            },
+            "colour" : {
+                "function" : self.color,
+                "description" : "Obtain the color of a role on the server",
+                "roles" : ["@everyone"]

```

Comment:
This should be `"roles" : ["@active","@new","@inactive"]` because `@everyone` let the command be used in a DM. Since a DM doesn't have any roles this will crash the extension.